applications/nrf_desktop/configuration/nrf52840_pca10056/app_ZDebug.conf

```diff
@@ -39,7 +39,6 @@ CONFIG_DESKTOP_BLE_SHORT_NAME="Mouse nRF52"
 CONFIG_DESKTOP_BLE_SECURITY_FAIL_TIMEOUT_S=10
 
 CONFIG_DESKTOP_CONFIG_ENABLE=y
-
```

Comment:
No need to remove empty line

Reply:
Please fix also other changes to empty lines which are not important for the PR.